planetarypraxis/smartforests
# Pick a featured radio episode and it will auto-appear on all pages

Closes #173

<img width="1133" alt="Screenshot 2022-03-09 at 11 28 15" src="https://user-images.githubusercontent.com/237556/157433032-cb0d1fd6-1206-4e4e-8229-abaf493bb91c.png">

## How Can It Be Tested?

1. Migrate
2. Go to the site and you should NOT see a player load up
3. Go to http://localhost:8000/admin/settings/smartforests/featuredcontent and pick a radio episode
4. Go to the site and you should see the player load up

## Types of changes


## Checklist:
